tools/src/main/scala/caliban/tools/ClientWriter.scala

```diff
@@ -195,9 +195,21 @@ object ClientWriter {
     val deprecated = field.directives.find(_.name == "deprecated") match {
       case None => ""
       case Some(directive) =>
-        s"@deprecated(${directive.arguments.collectFirst {
-          case ("reason", StringValue(reason)) => reason
-        }.fold(""""",""""")(r => s""""$r",""""")})\n"
+        val body =
+          directive.arguments.collectFirst {
+            case ("reason", StringValue(reason)) => reason
+          }
+
+        val tripleQuotes = "\"\"\""
```

Comment:
nitpick: you can move those 2 vals out of this function to avoid creating new ones each time.